pages/api/admin/users.js

```diff
@@ -0,0 +1,35 @@
+import connectMongo from "@config/mongo";
+import logger from "@config/logger";
+import { Profile } from "@models/index";
+
+export default async function handler(req, res) {
+  if (req.method != "GET") {
+    return res
+      .status(400)
+      .json({ error: "Invalid request: GET request required" });
+  }
+
```

Comment:
The API needs to be protected and only accessed by `admin` users. There is an example in the other admin api https://github.com/EddieHubCommunity/LinkFree/blob/main/pages/api/admin/events.js#L21